sources/packages/api/src/services/restriction/constants.ts

```diff
@@ -9,3 +9,24 @@ export const RESTRICTION_FEDERAL_MESSAGE =
  */
 export const RESTRICTION_PROVINCIAL_MESSAGE =
   "Your account has a hold on it. Please contact SABC to resolve.";
+/**
+ * Restriction code that are used in API.
+ * * The Restriction table will have more restriction
+ * * code, other than below mentioned codes.
+ */
+export enum RestrictionCode {
```

Comment:
I would say that we have been creating the enum more in plural, like `RestrictionCodes`.

Reply:
mm, while creating I check some recently created enum and looked at our wiki, it was singular 
![image](https://user-images.githubusercontent.com/77353155/172712907-4b09622e-7d5c-4334-ab13-c687e201ac77.png)
![image](https://user-images.githubusercontent.com/77353155/172712942-fae971b8-d861-4d3e-93af-1c6c3e4e03b3.png)
. i can update, if we are following plural

Reply:
Not clear which one we are using. Since there are both ways it is not a blocker.